code/game/objects/items/fireaxe.dm

```diff
@@ -40,6 +40,10 @@
 		bonus_modifier = 0 , \
 		butcher_sound = hitsound, \
 	)
+	var/datum/callback/door_start_trap_callback = CALLBACK(src, PROC_REF(start_trapping_door))
+	var/datum/callback/airlock_tip_callback = CALLBACK(src, PROC_REF(on_airlock_tip))
+
+	AddComponent(/datum/component/airlock_tip, 1 MINUTES, door_start_trap_callback, airlock_tip_callback)
```

Comment:
also this prevents bashing doors down with a fireaxe

Reply:
Would this also interfere with the crowbarring aspect of fire axes?